### PR TITLE
PAINTROID-402 Enabled ViewBinding

### PR DESCRIPTION
**Pull request by Smarshal21**
Enabled ViewBinding Successfully
Jira ticket - https://jira.catrob.at/browse/PAINTROID-402

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [ ] Include the name of the Jira ticket in the PR’s title
- [ ] Include a summary of the changes plus the relevant context
- [ ] Choose the proper base branch (*develop*)
- [ ] Confirm that the changes follow the project’s coding guidelines
- [ ] Verify that the changes generate no compiler or linter warnings
- [ ] Perform a self-review of the changes
- [ ] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [ ] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [ ] Stick to the project’s gitflow workflow
- [ ] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Comment by Smarshal21** [comment]
@exland @Lenkomotive @ThorstenBandel @StefanKutschera Could you please review this PR
and let me know the changes and i have created a new pr resolving the previous merge conflicts

**Comment by ThorstenBandel** [comment]
Hi, 
as this ticket is still in "Issues Pool" we have quite a few other tickets that we have to take a look at first. Especially at the moment we have a lot of requests and we are only a small team, so it will take some time to review the Pull Requests coming in. 
But we will take a look as soon as we get to it. There is no need to mention us on every PR we check all of them  regularly :)
BR
Thorsten

File: Paintroid/src/main/java/org/catrobat/paintroid/WelcomeActivity.kt [hunk 1]
```diff
@@ -75,29 +84,35 @@ class WelcomeActivity : AppCompatActivity() {
             }
 
             override fun onPageScrollStateChanged(state: Int) {
+                binding = PocketpaintSlideIntroWelcomeBinding.inflate(layoutInflater)
+                bindingw=  ActivityPocketpaintWelcomeBinding.inflate(layoutInflater)
+                bindingt = PocketpaintLayoutTopBarBinding.inflate(layoutInflater)
```
Comment (juliajulie95) [inline Paintroid/src/main/java/org/catrobat/paintroid/WelcomeActivity.kt:89]: formatting issue and please use better names

File: Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/DefaultSprayToolOptionsView.kt [hunk 2]
```diff
@@ -17,8 +17,59 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 package org.catrobat.paintroid.ui.tools
+//
```
Comment (juliajulie95) [inline Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/DefaultSprayToolOptionsView.kt:20]: Why is this commented ?

File: Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/DefaultTextToolOptionsView.kt [hunk 3]
```diff
@@ -33,13 +33,15 @@ import androidx.recyclerview.widget.LinearLayoutManager
...
 import org.catrobat.paintroid.R
+import org.catrobat.paintroid.databinding.DialogPocketpaintTextToolBinding
 import org.catrobat.paintroid.tools.FontType
 import org.catrobat.paintroid.tools.options.TextToolOptionsView
 
 private const val DEFAULT_TEXTSIZE = "20"
 private const val MAX_TEXTSIZE = "300"
 private const val MIN_FONT_SIZE = 1
 private const val MAX_FONT_SIZE = 300
+//private lateinit var binding:De
```
Comment (juliajulie95) [inline Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/DefaultTextToolOptionsView.kt:44]: ?

**Comment by juliajulie95** [comment]
Please check your variable names and formatting and reopen after fixing that

**Pull request closed without merging**